src/components/UpdateDeposit.re

```diff
@@ -120,14 +120,14 @@ let make = (~gorilla: Gorilla.gorilla) => {
 
   let onUnlockMetamaskAndOpenModal = event => {
     ReactEvent.Form.preventDefault(event);
-
+    ReactEvent.Form.stopPropagation(event);
     setModalOpen(_ => true);
   };
 
   <React.Fragment>
     <Rimble.Box p=2>
       <Rimble.Button onClick=onUnlockMetamaskAndOpenModal>
-        {React.string("Add/Remove Deposit")}
+        {React.string("Deposit")}
```

Comment:
This is less clear that you can remove deposit now

Reply:
Agreed. I will make an issue and have a tooltip that says the full word when you hover over it!

Nice catch :)

The reason I can't put it on the button is that when we have 3 gorillas next to each other all those words don't fit on the same line.

Reply:
Happy with that. Thanks for linking to the issue